adb_graphics/datahandler/gribfile.py

```diff
@@ -213,3 +213,52 @@ def open_kwargs(self):
             engine='pynio',
             lock=False,
             )
+
```

Comment:
not sure why the variable_names method is in here but not in main, I thought I was up to date on pulls.  I can remove it if it shouldn't be there.

Reply:
Yeah, I think this method should be removed. Maybe check to see if it breaks anything when running a few examples?

Reply:
Seems to be fine without this method. Leaving it out to look like the main branch.